addons/point_of_sale/static/tests/tours/utils/product_screen_util.js

```diff
@@ -660,3 +660,27 @@ export function addDiscount(discount) {
             .flatMap((key) => Numpad.click(key)),
     ].flat();
 }
+
+export function checkVariantProductInfo(titel) {
+    //check if title and price and inventory section are present
+    return [
+        {
+            content: "Check if modal title contains title",
+            trigger: ".modal-title.text-break.flex-grow-1:contains('Office Combo')",
+            run: function () {
+                const titleElement = document.querySelector(".modal-title.text-break.flex-grow-1");
+                const titleText = titleElement.textContent.trim();
+                return titleText.includes(titel);
+            },
+        },
+        {
+            content: "Check if the inventory section is present",
+            trigger: ".section-inventory.mt-3.mb-4.pb-4.border-bottom.text-start",
```

Comment:
Maybe `.section-inventory` is enough? The other classes are mostly for styling, so relying on them migh be a bit error prone? (like if we changed text-start to something else, the `.section-inventory` is still there, but this selector will fail since it didn't find `.text-start`).

(This comment is valid only if there are no other elements with the same class `section-inventory`)

```suggestion
            trigger: ".section-inventory.mt-3.mb-4.pb-4.border-bottom.text-start",
```